docs/maps/maps-aggregations.asciidoc

```diff
@@ -33,18 +35,28 @@ image::maps/images/grid_to_docs.gif[]
 
 [role="xpack"]
 [[maps-grid-aggregation]]
-=== Grid aggregation
+=== Clusters
 
-The *Grid aggregation* source uses {ref}/search-aggregations-bucket-geotilegrid-aggregation.html[GeoTile grid aggregation] to group your documents into grids. You can calculate metrics for each gridded cell.
+*Clusters* uses {ref}/search-aggregations-bucket-geotilegrid-aggregation.html[GeoTile grid aggregation] to group your documents into grids. You can calculate metrics for each gridded cell.
```

Comment:
I'm not sure if we need to make this change. While it removes some jargon, it introduces some changes on how things were named (e.g. would have to rename clusters->circles).

I'm also not sure if "heatmap" should be considered a cluster. It's more of a surface to give an indication of density/distribution of data.